onboarding/managers.md

```diff
@@ -7,43 +7,33 @@ description: Things for managers to remember during the early days of onboarding
 
 ### Before Day 1
 
-- [ ] Choose a peer mentor for the new hire
-- [ ] Choose a spirit guide for the new hire (consult with People Ops)
-- [ ] Create
-      [an onboarding github issue](https://github.com/artsy/potential/issues/new?template=engineering-onboarding.md)🔒
-      based on the onboarding template. Also invite the team to say hi, as this is a place that the new hire should
-      always be able to find easily in the future.
-- [ ] Set first-week schedule and let the new hire know their first day schedule on their private email account, as
-      there will be a period where their machine is still being setup and they won’t yet have access to their
-      calendar.
+- [ ] Choose an [onboarding mentor](/onboarding/mentors.md#the-onboarding-mentor) for the new hire. 
+- [ ] Create [an onboarding github issue](https://github.com/artsy/potential/issues/new?template=engineering-onboarding.md) based on the onboarding template. Tag the onboarding mentor and label the issue as `onboarding`. Also invite the team to say hi, as this is a place that the new hire should always be able to find easily in the future.
+- [ ] Set first-week schedule
 - [ ] Invite new hire to all engineering-wide meetings
-- [ ] Determine [sprint rotation plan](/onboarding/sprint-rotation.md) and update the [onboarding calendar](https://calendar.google.com/calendar/b/1?cid=YXJ0c3ltYWlsLmNvbV9jc2RocTFnNHNyMmhhY2Nvc3RmMTA1bTEwc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t)🔒 accordingly
+- [ ] Determine [sprint rotation plan](/onboarding/sprint-rotation.md) and update the [onboarding calendar](https://www.notion.so/artsy/Onboarding-Rotations-073f98b82d4840aaa0b2bba71343e7cc)🔒 accordingly
 
 ### Day 1
 
-- [ ] Organize lunch outing (probably to Tataki!)
-- [ ] Give an overview of what the onboarding will look like. This is a good time to walk through the onboarding
-      template.
+- [ ] Organize lunch outing, if applicable
+- [ ] Ask the new hire for their GitHub account. One can use their personal account. Invite them to Artsy Git and the Engineering and Engineering Staff teams.
+- [ ] Set up their [AWS account](https://www.notion.so/artsy/Add-a-new-AWS-user-c140d2ed914d4886a6359ff7f381ff0b)🔒. For consistency, it is encouraged to use the same name as their GitHub username.
+- [ ] Invite them to [DataDog](https://app.datadoghq.com/organization-settings/users/invite)🔒
+- [ ] Invite the new starter to [OpsGenie](https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/members)🔒. Add the engineer to the [On Call calendar](https://artsy.app.opsgenie.com/settings/schedule/detail/aa7d47fa-ab67-4c09-b9ca-d43a20efbb04)🔒 in either rot-a (US based) or rot-b (EU based).
```

Comment:
[The instructions for adding an engineer to the on-call schedule](https://github.com/artsy/README/blob/main/playbooks/incident-handling.md#on-call-rotations) are a little complex, so I suggest linking to them here rather than just the Opsgenie schedule.